llm_on_ray/inference/vllm_predictor.py

```diff
@@ -25,20 +26,25 @@
 
 
 class VllmPredictor(Predictor):
+    VLLM_CPU_KVCACHE_SPACE_DEFAULT = 40
+
     def __init__(self, infer_conf: InferenceConfig, max_num_seqs):
         super().__init__(infer_conf)
 
         model_desc = infer_conf.model_description
         model_config = model_desc.config
         dtype = "bfloat16" if infer_conf.vllm.precision == PRECISION_BF16 else "float32"
 
+        # Set environment variable VLLM_CPU_KVCACHE_SPACE to control the size of the CPU key-value cache.
+        # The default value is 40GB.
+        os.environ["VLLM_CPU_KVCACHE_SPACE"] = str(self.VLLM_CPU_KVCACHE_SPACE_DEFAULT)
```

Comment:
We should expose this config in the yaml file

Reply:
Could we do this later as we will refactor the config file, 40GB works for most of the scenario now.

Reply:
yes, that will be fine.